app/services/text_messenger.rb

```diff
@@ -0,0 +1,13 @@
+class TextMessenger
+  TWILIO_CLIENT = Twilio::REST::Client.new
+
+  def self.send_text(task)
+    phone = task&.project&.phone
+    return unless phone
+    TWILIO_CLIENT.messages.create(
+      from: '+13238941622', # my default valid from number for test acct
+      to: phone,
+      body: "Task complete!\n#{task.name}"
+    )
+  end
```

Comment:
It might be worth making this messenger less knowledgable about ` Task` as it's not overly relevant to what it is doing. It could perhaps take two parameters, a `phone` and a `name`.
Knowing it gets some thing that evidently might be `nil` based on your protective guarding and that thing may or may not have a project which may or may not have a phone seems like this simple thing that knows how to send a text message knows a lot about the internals of other objects.